src/dosbox.cpp

```diff
@@ -961,6 +961,10 @@ void DOSBOX_Init() {
 	                  "A single number is treated as the major version.\n"
 	                  "Common settings are 3.3, 5.0, 6.22, and 7.1.");
 
+	pint = secprop->Add_int("country", when_idle, 0);
+	pint->Set_help("Set DOS country code which will affect country-specific\n"
+	               "information such as date, time, and decimal formats.");
+
```

Comment:
Is there a way we could allow using country code letters here? There's probably plenty of people (me included!) who don't have a clue what country number they are.

Reply:
Thanks for the suggestion. I think it may be possible to make a mapping table to somehow convert keyboard layouts to country codes. I do not think it will be an exact conversation, but may probably help.

Reply:
A country mapping table is now available.

Reply:
Yay. Thank you.